src/inputs/internal/SelectFieldBase.tsx

```diff
@@ -69,133 +137,28 @@ export function SelectFieldBase<O, V extends Value>(props: SelectFieldBaseProps<
     };
   }
 
-  const [fieldState, setFieldState] = useState<FieldState>(initFieldState);
+  const [fieldState, setFieldState] = useState<FieldState<O>>(initFieldState);
 
   // Ensure we reset if the field's values change
   useEffect(() => setFieldState(initFieldState), [values]);
 
-  return (
-    <ComboBox<O, Key>
-      {...beamSelectFieldBaseProps}
-      multiselect={multiselect}
-      filteredOptions={fieldState.filteredOptions}
-      inputValue={fieldState.inputValue}
-      selectedKeys={fieldState.selectedKeys}
-      selectedOptions={fieldState.selectedOptions}
-      getOptionLabel={getOptionLabel}
-      getOptionValue={(o) => valueToKey(getOptionValue(o))}
-      onSelectionChange={(keys) => {
```

Comment:
I'm not sure how to really read this diff, but basically all of the props + functions here that were based to `ComboBox` are now just declared / mapped inline because `SelectFieldBase` _is_ `ComboBox`.